docs/news/whats_new.md

```diff
@@ -8,6 +8,32 @@ description: You can explore what's new in DHTMLX Kanban and its release history
 
 If you are updating Kanban from an older version, check [Migration to newer versions](news/migration.md) for details.
 
+## Version 2.0
+
+Released on Month Day, 2024
+
+### New functionality
+
+- An ability to display the editor as a modal window via the [`editor.placement`](../../api/config/js_kanban_editor_config) property
+- An ability to enable voting for a card (using the vote icon on the card) via the [`cardShape.votes.clicable`](../../api/config/js_kanban_columnshape_config) property
```

Comment:
mb this is simpler said as "A vote button on a card that can be added via blah-blah"